README.md

```diff
@@ -26,34 +26,47 @@
 
 **nf-core/references** is a bioinformatics pipeline that build references, for multiple use cases.
 
-It is primarily designed to build references for common organisms and store it on [AWS iGenomes](https://github.com/ewels/AWS-iGenomes/).
+For most common organisms the plan is to build the references and store them in the cloud, similar to [AWS iGenomes](https://github.com/ewels/AWS-iGenomes/).
```

Comment:
```suggestion
For most common organisms references will be stored them in the cloud, similar to [AWS iGenomes](https://github.com/ewels/AWS-iGenomes/), for XXXXXXXREASONXXXXXXXX.
```

Add a reason/'selling' point, and be more forceful 'the plan is' doesn't sound very polished or finished

Reply:
Updated